greenplumpython/table.py

```diff
@@ -96,6 +96,32 @@ def top(self, count: int, order_by: Iterable, skip: int = 0):
             parents=[self],
         )
 
+    def union(
+        self,
+        other: "Table",
+        my_targets: List = [],
+        other_targets: List = [],
```

Comment:
We can have some SQL like this:
`SELECT sex, UNNEST(col_nm) AS col_nm, UNNEST(coef) AS coef 
FROM plc_linreg_fitted
UNION 
SELECT sex, 'intercept' AS col_nm, intercept AS coef 
FROM plc_linreg_fitted;`
and to do so, with greenplumpython
`plc_linreg_fitted.union(
                plc_linreg_fitted,
                my_targets = [
                        plc_linreg_fitted["sex"],
                        unnest(plc_linreg_fitted["col_nm"]).rename("col_nm"),
                        unnest(plc_linreg_fitted["coef"]).rename("coef"),
                ],
                other_targets = [
                        plc_linreg_fitted["sex"],
                        "'intercept' AS col_name",
                        plc_linreg_fitted["intercept"].rename("coef"),
                ]
        )`

Reply:
So we remove targets list for both table from union function?

Reply:
Well, I think either we keep both, so the user can customize all with union function because it can be considered as one command when executing SQL command. Or we remove both for better reading, but users must to give extra names to save them.

Or maybe we can keep both scenarios, because with current version, we can also break the complex statement to several simple statements. 
```
plc_linreg_fitted_1 = plc_linreg_fitted[[
                        plc_linreg_fitted["sex"],
                        unnest(plc_linreg_fitted["col_nm"]).rename("col_nm"),
                        unnest(plc_linreg_fitted["coef"]).rename("coef")]
                ]
plc_linreg_fitted_2 = plc_linreg_fitted[[
                        plc_linreg_fitted["sex"],
                        "'intercept' AS col_name",
                        plc_linreg_fitted["intercept"].rename("coef")]
                ]
plc_linreg_fitted_1.union(plc_linreg_fitted_2)
```
This works